.github/workflows/deploy-beta.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '8.2'  # musí odpovídat verzi na wedosu
+          php-version: '8.3'  # musí odpovídat verzi na serveru
```

Comment:
Tohle by se nám stávat nemělo - že v produkci máme novější verzi jazyka než pro vývoj a pro testy.

Z praxe vím, že občas nějaký zdánlivě neškodný deprecated dokáže při dobře míněném upgrade verze PHP sestřelit aplikaci.

Reply:
ano, to by se stavat asi nemelo; produkce je momentalne 8.3